### PR TITLE
add and fix shields in Readmes

### PR DESCRIPTION
**Pull request by ameknite**
# Objective

Fix shields

## Solution

- Correct shield in the Bevy ECS Readme, where only the MIT license is displayed
Screenshot 2023-10-01 at 18 28 27
- Add shields to other Readmes.
- homogenize shields and titles.

### CODE REVIEW

**Comment by mockersf** [comment]
I don't think other readmes than the root one should have shields

**Comment by ameknite** [comment]
I always thought the purpose of dividing bevy into crates was so that you could use the crates separately. Is that not the case?

**Comment by alice-i-cecile** [comment]
> I don't think other readmes than the ones that can be used independently should have shields

Agreed. I want this for `bevy_ecs`, `bevy_reflect`, `bevy_tasks`, `bevy_utils` and `bevy_mikktspace`. Those are the only ones that can be used truly independently, all the others need `bevy_ecs` in order to function.

**Review by alice-i-cecile — APPROVED** [review]
Oh right, you could use `bevy_ptr` independently too. Your call on whether to add it there.

File: crates/bevy_ptr/README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# `bevy_ptr`
+# Bevy Ptr
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](https://github.com/bevyengine/bevy#license)
+[![Crates.io](https://img.shields.io/crates/v/bevy_ptr.svg)](https://crates.io/crates/bevy_ptr)
+[![Crates.io](https://img.shields.io/crates/d/bevy_ptr.svg)](https://crates.io/crates/bevy_ptr)
```
Comment (james7132) [inline crates/bevy_ptr/README.md:5]: Why is crates.io duplicated twice here? Should these be the docs?

**Review by james7132 — CHANGES_REQUESTED** [review]

File: crates/bevy_ptr/README.md (hunk 1 above)
Comment (ameknite) [reply crates/bevy_ptr/README.md:5]: I will change the text to Downloads and also add the Docs badge

**Review by james7132 — APPROVED** [review]

**Pull request merged**